middleware/errorMiddleware.js

```diff
@@ -0,0 +1,16 @@
+const notFound = (req, res, next) => {
+    const error = new Error(`Not Found - ${req.originalUrl}`);
+    res.status(404);
+    next(error);
+  };
+  
+  const errorHandler = (err, req, res, next) => {
+    const statusCode = res.statusCode === 200 ? 500 : res.statusCode;
```

Comment:
if statusCode is 200 we are returning 500, is this logic correct?

Reply:
I will change this logic thanks for update.